recipes/pastrami/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+#chmod +x $SRC_DIR/pastrami.py
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+$PYTHON -m pip install --no-deps .  
+PIP_NO_CACHE_DIR=True
+$PYTHON -m pip install --no-deps .
```

Comment:
This runs the installation three times, which should not be necessary. I would suggest that you get rid of `build.sh` entirely and just put `script: "{{ PYTHON }} -m pip install . --no-deps -vv"` into the `build:` section of the recipe.

Reply:
Done